portfolio/src/main/java/com/google/sps/data/CommentBuilder.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class CommentBuilder {
+    public CommentBuilder() {
+        this.comment = new Comment();
+    }
+
+    public CommentBuilder setId(int id) {
+        this.comment.setId(id);
+
+        return this;
+    }
+
+    public CommentBuilder setName(String name) {
+        this.comment.setName(name);
+
+        return this;
+    }
+
+    public CommentBuilder setDescription(String description) {
+        this.comment.setDescription(description);
+
+        return this;
+    }
+
+    public CommentBuilder setPostDate(Date postDate) {
+        this.comment.setPostDate(postDate);
+
+        return this;
+    }
+
+    private void resetProcess() {
+        this.comment = new Comment();
+    }
+
+    public Comment build() {
```

Comment:
Up to you here but if you remove the reset process then you can easily build multiple of the same or very similar comments. This will be useful in tests and the like

Reply:
Can also have build perform the check that all the fields have been set